google/ggrc-core
# QUICK-FIX Update license headers

# Issue description

The license header script had a few bugs and it checked for too many files.

# Steps to test the changes

- Remove the license header from the license header script and check the results.
- Add gif or svg or some binary to our codebase and run the script

# Solution description

Refactor the grep to not contain the exact string it's grepping.
Remove check for existing binary files in our codebase.


# Sanity checklist




# PR Review checklist
